apps/system/test/marionette/browser_clear_browsing_history_test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+var Server = require('../../../../shared/test/integration/server');
+var Rocketbar = require('./lib/rocketbar');
+var Settings = require('../../../settings/test/marionette/app/app');
+
+var assert = require('assert');
+
+marionette('Browser test', function() {
+
+  var client = marionette.client({
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var search, system, server, home, rocketbar, settingsApp, actions;
+
+  suiteSetup(function(done) {
+    Server.create(__dirname + '/fixtures/', function(err, _server) {
+      server = _server;
+      done();
+    });
+  });
+
+  suiteTeardown(function() {
+    server.stop();
+  });
+
+  setup(function() {
+    home = client.loader.getAppClass('homescreen');
+    search = client.loader.getAppClass('search');
+    system = client.loader.getAppClass('system');
+    rocketbar = new Rocketbar(client);
+    settingsApp = new Settings(client);
+    actions = client.loader.getActions();
+    system.waitForFullyLoaded();
+  });
+
+  test('Clear browsing history', function() {
+    // Open then close web page.
+    var url = server.url('sample.html');
+    rocketbar.homescreenFocus();
```

Comment:
You can use rocketbar.goToURL(url), which does exactly the same than these 3 lines.